games/__init__.py

```diff
@@ -4,9 +4,10 @@
 
 
 games = {
+
     'ttt': Game(
         name='Tic-Tac-Toe',
-        desc='Achieve three in a row horizontally, vertically, or diagonally',
+        desc='Place your piece onto an open square. Wins when you connect three in a row horizontally, vertically, or diagonally',
```

Comment:
Nit: What do you think if we make all game/variant descriptions as sentences so they all end with a sentence period (`.`)?